test/testlocal/nft/anvil/NFT.t.sol

```diff
@@ -233,8 +233,8 @@ contract NFTAnvil is AnvilFork {
    * test locking collateral
    */
 
-  function test_allowSAFE(uint256 cTypeIndex, uint8 ok) public {
-    vm.assume(ok < 2);
+  function test_allowSAFE(uint256 cTypeIndex, bool ok) public {
+    // vm.assume(ok < 2);
```

Comment:
Can you update this check for the new bool type?

Reply:
Sure.

Reply:
I'll push up the change - although I am curious why we would make this change during the AccountingEngine test fixes?